openedx/adg/lms/webinars/models.py

```diff
@@ -319,6 +321,7 @@ class WebinarRegistration(TimeStampedModel):
     week_before_mandrill_reminder_id = models.CharField(
         default='', max_length=255, verbose_name=_('Scheduled Week Before Reminder Id On Mandrill'),
     )
+    zoom_id = models.CharField(verbose_name=_('User Zoom Id'), max_length=100, blank=True, null=True, )
```

Comment:
Shouldn't the variable be named `user_zoom_id`? Because zoom id is confusing and mixing with the zoom_id of the webinar. If the user zoom id and webinar zoom id are different, they should be named differently.